src/test/kotlin/io/bazel/kotlin/builder/tasks/BUILD.bazel

```diff
@@ -107,6 +107,12 @@ kt_rules_test(
     ],
 )
 
+kt_rules_e2e_test(
```

Comment:
You probably want `kt_rules_test` here instead.

Reply:
moved this to assertion tests.